consensus/ibft/src/main/java/org/hyperledger/besu/consensus/ibft/validation/MessageValidator.java

```diff
@@ -47,19 +48,19 @@ public class MessageValidator {
    *
    * @param signedDataValidator the signed data validator
    * @param proposalConsistencyValidator the proposal consistency validator
-   * @param blockValidator the block validator
+   * @param protocolSchedule the proposal schedule
```

Comment:
javadoc for this should be "the protocol schedule"

Reply:
🤦 - fixed